index.html

```diff
@@ -198,6 +198,30 @@ <h3>Expression Format</h3>
           [=preference=] for the Global Privacy Control value.
         </p>
       </section>
+      <section>
+        <h3>Preference Caching</h3>
+        <p>
+          The [=preference=] MUST be cached on every top-level navigation to ensure consistency in communication of
+          the person's request that their data "not be sold or shared." This means that if the [=preference=] changes
+          during or after a top-level navigation, it WILL NOT be reflected until the next navigation.
+        </p>
+        <p>
+          A [=top-level browsing context=] has a <dfn><code>gpcAtNavigation</code></dfn> boolean.
+          It is initially <code>false</code>.
+        </p>
+        <p>
+          When [=process a navigate fetch=] is run with [=top-level browsing context=] as <code>browsingContext</code>,
+          [=top-level browsing context=]'s <dfn><code>gpcAtNavigation</code></dfn> is updated to
+          <code>true</code> if the [=preference=] is enabled, and <code>false</code> if the the [=preference=] is disabled
+          or defaulted to.
+        </p>
+        <p>
+          If [=preference=] is changed to be inconsistent with some <code>gpcAtNavigation</code> cached in a
+          [=top-level browsing context=], the user agent SHOULD prompt for any inconsistent tabs to be reloaded
+          (refreshing the cached <code>gpcAtNavigation</code> to reflect the current [=preference=]). The user agent
+          MAY prompt for a refresh of all inconsistent tabs at once or for each individually as the user interacts with them.
```

Comment:
I think we could be less specific about the UA behavior here.

Maybe:
> the user agent SHOULD inform the user of any inconsistent tabs and provide the option to reload them, refreshing the cached gpcAtNavigation to reflect the current preference.

(Personally, if I have dozens of tabs open and change my settings, I probably don't want to refresh them all, or see a refresh prompt every time I interact with a tab. But I will want to know that my settings won't take effect until I refresh, and I might choose to refresh one or all. Browsers can reasonably provide different UI options here, but I don't think we need to specify just these two refresh prompt options.)